File: BeeCrowd/DiamondsAndSand.java
import java.util.Scanner;

public class DiamondsAndSand {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int testes = sc.nextInt();
        sc.nextLine();
    
        for(int i = 0; i < testes; i++){
            int contador = 0;
            int abrirChaves = 0;
            String entrada = sc.nextLine();

            for(int j = 0; j < entrada.length(); j++){
                 if (entrada.charAt(j) == '<'){
                    abrirChaves++;
                }else if((entrada.charAt(j) == '>' && abrirChaves > 0)){
                    contador++;
                    abrirChaves--;
                }
            }
            
            if(contador > 0){
                System.out.println(contador);
            } else {
                System.out.println(0);
            }
        }

        sc.close();
    }
}

File: BeeCrowd/DiamondsAndSand.c
#include <stdio.h>
#include <string.h>

int main(){
    int testes;
    scanf("%d", &testes);
    getchar();

    char entrada[100];

    for(int i = 0; i < testes; i++){
        int abrirChaves = 0, contador = 0;
        
        fgets(entrada, sizeof(entrada), stdin);
        
        for(int j = 0; j < strlen(entrada); j++){
            if(entrada[j] == '<'){
                abrirChaves++;
            } else if(entrada[j] == '>' && abrirChaves > 0){
                contador++;
                abrirChaves--;
            }
        }

        printf("%d\n", contador);
    }

    return 0;
}